layouts/course/section.html

```diff
@@ -20,7 +20,7 @@ <h1 class="no-margin">{{.Title}}</h1>
         <h3>chapters</h3>
         <ul>
             {{ range ( where .Sections "Type" "course_chapter" ) }}
-            <li><a href="#{{ replace .Params.title " " "-" }}"> {{ .Params.title }} </a></li>
+            <li><a href="{{ .Permalink }}"> {{ .Params.title }} </a></li>
```

Comment:
This linked to the headings for each chapter on the page. That was mostly for the Krita course and its 40+ videos.
The chapter list pages shouldn't even exist.

Reply:
Right, that's what I thought initially, but it's probably that these headings got removed with the latest changes, cause it they don't link to anything at the moment.

About removing the chapter list pages. I'm not sure how that would work with the `.Pages` variable that we use now. We kind of depend on this list page to get to `.RegularPages` in the inner `range` loop.

Reply:
I don't mean removing the data but rather finding a way to not have them on the final website maybe, or at least to exclude them from search results. You know, the search on the docs search everything right now, and if we add a search bar to the website for similar instant search (that'd be great), I'd like us to exclude all the pages we have that aren't so useful.